json.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
Filename does not seem accurate.

Reply:
I think it does. It defines the structs used to load JSON form RPC responses.
What would you call it?